src/overlays/actors/ovl_Obj_Purify/z_obj_purify.c

```diff
@@ -27,42 +56,256 @@ const ActorInit Obj_Purify_InitVars = {
     (ActorFunc)NULL,
 };
 
-#endif
+ObjPurifyInfo ObjPurifyInfoList[] = {
+    {
+        OBJECT_NUMA_OBJ,
+        0.1f,
+        { NULL, NULL },
+        { object_numa_obj_DL_0128E0, object_numa_obj_DL_012D90 },
+        { object_numa_obj_Matanimheader_012CF8, object_numa_obj_Matanimheader_0131A8 },
+        &object_numa_obj_Colheader_012818,
+        0,
```

Comment:
```suggestion
        false,
```
The last element of the struct looks to be boolean, so should use `false/true`. Applies to all elements of the array.